### PR TITLE
Bitrise step appdome build 2secure android 1.0.2

### PR DESCRIPTION
**Pull request by elish266**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] __I will not move an already shared step version's tag to another commit__
- [ ] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [ ] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [ ] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [ ] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

**Comment by vshah23** [comment]
Hi @elish266 we appreciate your contribution to the step library. I noticed you have included step definition changes for the iOS step in the PR in addition to the Android changes. Will you please open a separate PR per step?

File: steps/bitrise-step-appdome-build-2secure-android/1.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+title: Appdome-Build-2Secure step for Android
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API. For detailes see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
+website: https://github.com/Appdome/bitrise_build-2secure-android
```
Comment (vshah23) [inline steps/bitrise-step-appdome-build-2secure-android/1.0.2/step.yml:6]: Is this the correct URL? I noticed the git source url goes to a different repository below.
`https://github.com/Appdome/bitrise-step-appdome-build-2secure-android.git`
vs
`https://github.com/Appdome/bitrise_build-2secure-android`

Comment (elish266) [reply steps/bitrise-step-appdome-build-2secure-android/1.0.2/step.yml:6]: That's correct.

File: steps/bitrise-step-appdome-build-2secure-android/1.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+title: Appdome-Build-2Secure step for Android
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API. For detailes see: https://www.appdome.com/how-to/appsec-release-orchestration/mobile-appsec-cicd/use-appdome-build-2secure-step-for-bitrise
```
[the remark this answers is not in the document]
Comment (elish266) [reply steps/bitrise-step-appdome-build-2secure-android/1.0.2/step.yml]: The URL will be avialable after our documentation is finalized. It's in final review stages now.

**Pull request closed without merging**